### PR TITLE
[RFR]Select by field values

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,12 +179,24 @@ Allow to create a query to update one entity or create it if it does not already
...
  - table: the name of the table in which to upsert
  - selectorFields: the field used to select one entity checking if it exists
  - updatableFields: the field that can be updated
  - returnFields: the field to return in the result
  - fields: all the fields accepted by the query, default to selectorFields + updatableFields (no reason to change that)
+####parameters
+- entities: array of entities to upsert
+
+###selectByFieldValues(table, selectorField, returnFields)(db)(values)
+Allow to create a query to select ann entity with selectorField IN values and keep the ORDER of values.
```
Comment (alexisjanvier) [inline README.md]: sed/ann/an/

Comment (alexisjanvier) [inline README.md]: s/ann/an

Comment (alexisjanvier) [inline README.md]: s/ann/an/

**Comment by alexisjanvier** [comment]
It looks fine, but it's a very specific request !

File: README.md (hunk 1 above)
Comment (Kmaschta) [inline README.md]: victory

**Comment by ThieryMichel** [comment]
It's useful whe you have a many to many relationship for which the order is important.

**Comment by alexisjanvier** [comment]
👍

**Pull request merged**